### DISCUSSION TITLE
Polish translations

### DISCUSSION THREAD

**Discussion by Romek1234** [GrandOrgue/grandorgue#743] [General]
I'd like to add a brick to the Granorgue building.. I would like to create and add a file pl.po - Polish. translations for GrandOrgue package.

**Comment by oleg68** [comment]
Great. Please add a PR

**Reply by Romek1234** [reply]
OK! I added.

**Reply by larspalo** [reply]
Looks good, except for the line 7 which contain the mail address to the developers mailing list on sourceforge. How should we handle such bug report contact information here on GitHub?

**Reply by larspalo** [reply]
Also line 12, language should be "pl", right?

**Reply by Romek1234** [reply]
I downloaded the repository to a local disk and (compiled opensuse leap 15). Without adding pl.po and no changes ./po/LINGUAS compiles fine. After adding my ./po/pl.po file and adding pl w ./po/LINGUAS does not compile. What files should I change, add or translate to receive Polish language compilations?

**Reply by Anetka71** [reply]
I confirm, it doesn't compile for me either.

Such errors throws:
...
ependee "/home/romek/GO-src/build-for/linux/help/CMakeFiles/build-help.dir/DependInfo.cmake" is newer than depender "/home/romek/GO-src/build-for/linux/help/CMakeFiles/build-help.dir/depend.internal".
Dependee "/home/romek/GO-src/build-for/linux/help/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/romek/GO-src/build-for/linux/help/CMakeFiles/build-help.dir/depend.internal".
Scanning dependencies of target build-help
make[2]: Leaving directory '/home/romek/GO-src/build-for/linux'
make -f help/CMakeFiles/build-help.dir/build.make help/CMakeFiles/build-help.dir/build
make[2]: Entering directory '/home/romek/GO-src/build-for/linux'
make[2]: Nothing to be done for 'help/CMakeFiles/build-help.dir/build'.
make[2]: Leaving directory '/home/romek/GO-src/build-for/linux'
[ 34%] Built target build-help
make[1]: Target 'all' not remade because of errors.
make[1]: Leaving directory '/home/romek/GO-src/build-for/linux'
make: *** [Makefile:169: all] Error 2
make: Target 'package' not remade because of errors.

**Reply by Romek1234** [reply]
I wonder if pl is needed in the file /po/LINGUAS ?
What about @Oleg68 ?

**Comment by oleg68** [comment]
@Romek1234 @Anetka71 

Maybe the reason was syntax errors in pl.po. See #753

Please try to test the intermediate GitHub build from https://github.com/GrandOrgue/grandorgue/actions/runs/1307218898 It should include the polish translation.

**Comment by oleg68** [comment]
> I wonder if pl is needed in the file /po/LINGUAS

Seems po/LINGUAS is not used.

**Comment by Romek1234** [comment]
Unfortunately, this compilation https://github.com/GrandOrgue/grandorgue/actions/runs/1307218898 does not support Polish.

**Comment by oleg68** [comment]
After #757 I was able to switch language to polish.

**Reply by Romek1234** [reply]
Thank you very much!
Thanks a lot to all of you.

**Comment by Anetka71** [comment]
Oh, I can see there is a great job here! I have downloaded and tested another intermediate GrandOrgue version. Polish language support is already working. I think there will be a few more corrections to the translations. Many thanks to everyone.

**Reply by oleg68** [reply]
> I have downloaded and tested another intermediate GrandOrgue version. Polish language support is already working

Which version? 3.2.1-0.29?

**Reply by Romek1234** [reply]
Yes 3.2.1-0.29 .And 3.2.1-0.27

**Reply by Anetka71** [reply]
I have downloaded and am testing the version 3.2.1-0.27 for Windows10.
Tomorrow I will download and test 3.2.1-0.29.
